evm/logic/system.py

```diff
@@ -28,6 +30,16 @@ def return_op(computation):
     raise Halt('RETURN')
 
 
+def revert(computation):
+    start_position, size = computation.stack.pop(num_items=2, type_hint=constants.UINT256)
+
+    computation.extend_memory(start_position, size)
+
+    output = computation.memory.read(start_position, size)
+    computation.output = bytes(output)
+    raise Revert(bytes(output))
```

Comment:
Maybe move the `bytes()` call to line 38 so that you don't have to repeat it in 39 and 40?